.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @baywet @ddyett @MichaelMainer @nikithauc @zengin
```

Comment:
Q: should we remove the REVIEWERS from "create pull request" YAMLs to maintain the list in single place?

Reply:
Agreed.

Reply:
Although I am interested to see what happens if I leave them in.

Reply:
Removed.